common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3317,6 +3317,12 @@
     ],
     "sqlState" : "XXKST"
   },
+  "STREAMING_STATEFUL_OPERATOR_NOT_MATCH_IN_STATE_METADATA" : {
+    "message" : [
+      "Streaming stateful operator name does not match with the operator in state metadata with the same operator id (id: <operatorId>). Stateful Operator name for current batch: <currentOperatorName>; Operator name in the state metadata: <stateMetadataOperatorName>."
```

Comment:
Can we explain why this occur to the customer? Like "changing stateful operator of existing streaming query is not allowed."